### PR TITLE
converting to python3

### PR DESCRIPTION
**Pull request by hipposen**

### CODE REVIEW

File: plugins/fuel.py [hunk 1]
```diff
@@ -1,102 +1,74 @@
...
-        irc.registerCommand("!fuel", "Treibstoffpreise")
+    bot.log.info('Fetching fuel info for %s' % fuelsort)  
 
-    def _get_fuel_stations(self):
-        url = "https://creativecommons.tankerkoenig.de/json/list.php?" + \
+    url = "https://creativecommons.tankerkoenig.de/json/list.php?" + \
             "lat=50.9827792" + \
             "&lng=11.0394426" + \
-            "&rad=15" + \
+            "&rad=1" + \
```
Comment (mkzero) [inline plugins/fuel.py]: Any reason why this changed from 15 to 1?

File: plugins/fuel.py [hunk 2]
```diff
@@ -1,102 +1,74 @@
...
+import json
+import aiohttp
 
+@command(permission="view")
+@asyncio.coroutine
+def fuel(bot, mask, target, args):
+    """Show the current fuel price for fuelsort
 
-class fuel(Plugin):
+        %%fuel [<fuelsort>]...
```
Comment (mkzero) [inline plugins/fuel.py]: Maybe call that fueltype? `fuelsort` kinda sounds like a hot new sorting algorithm

**Comment by mkzero** [comment]
Apart from the few comments I made the code looks good. I'll manually test it later, but apart from testing this looks fine to merge.

File: plugins/fuel.py (hunk 2 above)
Comment (hipposen) [inline plugins/fuel.py]: :-) next version updating to more readable

**Comment by mape2k** [comment]
Closed, we'll use #153. Thank you for your contribution.

**Pull request closed without merging**